avm/utilities/pipelines/e2eValidation/resourceRemoval/helper/Get-DeploymentTargetResourceList.ps1

```diff
@@ -70,7 +70,7 @@ function Get-DeploymentTargetResourceListInner {
             } else {
                 # In case the resource group itself was already deleted, there is no need to try and fetch deployments from it
                 # In case we already have any such resources in the list, we should remove them
-                [array]$resultSet = $resultSet | Where-Object { $_ -notmatch "\/resourceGroups\/$resourceGroupName\/" }
+                return $resultSet | Where-Object { $_ -notmatch "\/resourceGroups\/$resourceGroupName\/" } | Select-Object -Unique
```

Comment:
Any chance we can combine all updates (I'm counting 3 PRs) to the removal logic into one to minimize the number of workflow runs?

Reply:
@AlexanderSehr nvm, as this one is blocking another PR while others are still in draft, I'll move this comment to the other PRs